src/status_im2/contexts/onboarding/common/overlay/style.cljs

```diff
@@ -1,5 +1,6 @@
 (ns status-im2.contexts.onboarding.common.overlay.style
```

Comment:
Maybe we should try to get rid of this namespace. The background component itself has a property to show a dark overlay, And I think we used that in the original implementation. If component don't support showing/hiding overlay as runtime, we should fix/implement that.

Also animation of navigation to "I'm new to Status" is different from [Figma](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=3672-369971&mode=design&t=47MYAAKGdvdrV9WK-0). (Not sure if they are updated recently)
(We maybe can get rid of this namespace while implementing figma animations)